api/Models/Game.cs

```diff
@@ -18,5 +18,6 @@ public class Game
 
         public List<DataPoint>? DataPoints { get; set; } = new List<DataPoint>();
         
+        public string? boxArtURL { get; set; } = string.Empty;
```

Comment:
For consistency we should capitalize the B on the api side for this

Reply:
OK 👍